apps/gateway/.env.example

```
@@ -6,6 +6,13 @@ PORT=3002
 STACK_SECRET_SERVER_KEY=your-stack_secret_server_key
 VITE_STACK_PROJECT_ID=your-stack_project_id
 
-# Model API Keys
+# LLM Auth
+AWS_ACCESS_KEY_ID=your-aws_access_key_id
+AWS_SECRET_ACCESS_KEY=your-secret_access_key
+
 GROQ_API_KEY=your-groq_api_key
+
+GOOGLE_VERTEX_SERVICE_ACCOUNT=your-google_vertex_service_account
+GOOGLE_VERTEX_PROJECT=your_vertex_project
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GOOGLE_VERTEX_PROJECT key should go before the GOOGLE_VERTEX_SERVICE_ACCOUNT key

(UnorderedKey)